test/CMakeLists.txt

```diff
@@ -1328,6 +1333,22 @@ if(iodaconv_bufr_ENABLED)
                             gdas.t12z.adpsfc.tm00.nc ${IODA_CONV_COMP_TOL_ZERO}
                     DEPENDS bufr2ioda.x )
 
+  ecbuild_add_test( TARGET  test_iodaconv_prepbufr_ncep_api_adpsfc2ioda
```

Comment:
At some point in this test definition, you need
ENVIRONMENT "PYTHONPATH=${IODACONV_PYTHONPATH}"
see some of the other tests in this files. That will eliminate the need for appending path.